tests/fnet.rs

```diff
@@ -227,7 +228,7 @@ fn fnet_for_token_classification() -> anyhow::Result<()> {
 
     //    Forward pass
     let model_output = no_grad(|| {
-        fnet_model
+        fnet_model.unwrap()
```

Comment:
can you please be consistent for all tests with the model error handling  (see the changes for `albert` for example):
`let albert_model = AlbertForTokenClassification::new(vs.root(), &config)?;`

Reply:
Done